mmdeploy/codebase/base/task.py

```diff
@@ -334,7 +334,7 @@ def get_postprocess(self) -> Dict:
         pass
 
     @abstractmethod
-    def get_model_name(self) -> str:
+    def get_model_name(self, *args, **kwargs) -> str:
```

Comment:
it seems no need to add this two arguments if all you need is in self.model_cfg.

Reply:
For OCR, `work_dir` is used. And we'd better add this since we don't know if we will add more args in the future.